src/commands/client-setinfo.json

```diff
@@ -0,0 +1,46 @@
+{
+    "SETINFO": {
+        "summary": "Set client or connection specific info",
+        "complexity": "O(1)",
+        "group": "connection",
+        "since": "7.2.0",
+        "arity": 4,
+        "container": "CLIENT",
+        "function": "clientSetinfoCommand",
+        "command_flags": [
+            "NOSCRIPT",
+            "LOADING",
+            "STALE",
+            "SENTINEL"
+        ],
+        "acl_categories": [
+            "CONNECTION"
+        ],
+        "reply_schema": {
+            "const": "OK"
```

Comment:
This can also return a string representing a status message.

Reply:
i think we eventually aborted that idea and said it'll just either succeed or fail (failure error message can be verbose, but we don't document these here).

Reply:
Okay, good to know I suppose. I believe the other `ACL DRYRUN` had a string status message, which is what I was comparing it to.